internal/provider/password_resource.go

```diff
@@ -184,6 +185,12 @@ func (r *passwordResource) Schema(ctx context.Context, req resource.SchemaReques
 				Description: "The display name for the password.",
 				Optional:    true,
 			},
+			"cidrs": schema.ListAttribute{
+				Description: "List of IP addresses or CIDR ranges that can use this password.",
+				Optional:    true,
+				Computed:    true,
```

Comment:
user input so it should not be computed - https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed